corehq/util/pagination.py

```diff
@@ -182,6 +195,31 @@ def _get_event_handler(self):
         else:
             return ResumableIteratorEventHandler(self)
 
+    def retry(self, item_id, max_retry=3):
+        """Add document to be yielded at end of iteration
+
+        Iteration order of retry documents is undefined. All retry
+        documents will be yielded after the initial non-retry phase of
+        iteration has completed, and every retry document will be
+        yielded each time the iterator is stopped and resumed during the
+        retry phase. This method is relatively inefficient because it
+        forces the iteration state to be saved to couch. If you find
```

Comment:
fwiw, changing it to save to "meta" has orphaned the status records for previous blob migrations. I doubt you care about that though.